pages/about.html

```diff
@@ -1,12 +1,90 @@
 <!DOCTYPE html>
```

Comment:
This is hands down my favorite page!! These memes!! 💯 💀 🔥 

Great job making the wall of memes in the background here!!